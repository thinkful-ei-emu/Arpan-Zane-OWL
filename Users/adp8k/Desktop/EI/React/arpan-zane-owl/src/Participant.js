import React from 'react';

function Participant(props){
    return (
        <div>
            <h2>{props.name}</h2>
            <img src={props.avatar} alt="avatar"/>
            <p>{props.inSession.toString()}</p>
            <p>{props.onStage.toString()}</p>
        </div>
    )
}

export default Participant;