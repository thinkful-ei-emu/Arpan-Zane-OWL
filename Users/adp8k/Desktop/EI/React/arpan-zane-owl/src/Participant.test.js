import React from 'react';
import ReactDOM from 'react-dom';
import Participant from './participant'
import renderer from 'react-test-renderer';


describe('<Participant />', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');

        ReactDOM.render(<Participant />, div);


        ReactDOM.unmountComponentAtNode(div);

    });

    it('renders this UI as expected', () => {
        // Render the component, as JSON
        const tree = renderer.create(<Participant />).toJSON();
        // Check whether it matches the previous snapshot
        // Stored in __snapshots__/App.test.js.snap
        expect(tree).toMatchSnapshot(); 
    });
});