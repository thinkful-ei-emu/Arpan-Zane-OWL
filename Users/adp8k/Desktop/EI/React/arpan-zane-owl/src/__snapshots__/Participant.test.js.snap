// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Participant /> renders this UI as expected 1`] = `
<div>
  <h2 />
  <img
    alt="avatar"
  />
  <p />
  <p />
</div>
`;
